### ISSUE TITLE
Feature Request: Input auto-focus on login page.

### ISSUE THREAD

**Issue by n8detar** [bbilly1/tubearchivist#104]
Right now on the login page, you have to select the username filed before typing. It would be nice to have this field automatically selected when the page loads. Adding the `autofocus` attribute to the `<input>` element for the username form should automatically apply the focus when the page loads. The `autocomplete` attribute might also be nice for the username. This will let the browser show options to fill in the field.
Ex:
```
<form action="/login/" method="POST" name="login">
     <input type="text" name="username" placeholder="Username" maxlength="150" id="id_username" required autofocus autocomplete>
     <br>
     <input type="password" name="password" placeholder="Password" id="id_password" required autocomplete>
     <button type="submit">Login</button>
</form>
```

**Comment by bbilly1** [comment]
Yes, good idea, no need to click into the form first. 

Thanks for your input!

**Comment by bbilly1** [comment]
Is now merged in v0.0.9. Thanks for your input.
